www/apps/book/app/learn/fundamentals/modules/db-operations/page.mdx

```diff
@@ -353,7 +353,78 @@ You can then use the data model repository in your service to perform database o
 
 A data model repository has methods that allows you to create, update, and delete records, among other operations.
 
-To learn about the methods available in a data model repository, refer to the [Data Model Repository](!resources!/data-model-repository-reference) reference.
+Here are examples of typical operations using a repository:
+
+```ts
+// Create a new product
+const product = await productRepository.create({
+  title: "Ergonomic Chair",
+  description: "Office chair with lumbar support",
+  price: 299.99,
+  inventory_quantity: 50
+})
+
+// Find a specific product
+const product = await productRepository.findOne({
+  where: { id: "prod_123456789" }
+})
+
+// Update a product
+await productRepository.update("prod_123456789", {
+  inventory_quantity: 42,
+  metadata: {
+    featured: true
+  }
+})
+
+// Delete a product
+await productRepository.delete("prod_123456789")
+
+// Find multiple products with filtering
+const products = await productRepository.find({
+  where: { 
+    status: "published", 
+    inventory_quantity: { $gt: 0 }
+  },
+  take: 10,
+  skip: 0,
+  order: { created_at: "DESC" }
+})
```

Comment:
Since the document constantly refers to a `post` model, can you use `postReopsitory` examples instead? You can use fields relevant to posts like `title` or `published_at`.